HISTORY.md

```diff
@@ -3,6 +3,9 @@
 ### Behavior Changes
 * Best-efforts recovery ignores CURRENT file completely. If CURRENT file is missing during recovery, best-efforts recovery still proceeds with MANIFEST file(s).
 * In best-efforts recovery, an error that is not Corruption or IOError::kNotFound or IOError::kPathNotFound will be overwritten silently. Fix this by checking all non-ok cases and return early.
+* When `file_checksum_gen_factory` is set to `GetFileChecksumGenCrc32cFactory()`, BackupEngine will calculate the crc32c checksum of the file to be backed up and compare the resulting value with the one stored in the DB manifest before copying the file to the backup directory. In addition, after copying to the backup directory, BackupEngine will double-check the checksum of the copied file with the one calculated before copying. This helps detect some rare corruption events such as bit-flips during the copying process.
```

Comment:
We may need to point out the case that if the file_checksum_gen_factory is not set or set to other customized factory.